app/assets/stylesheets/welcome.scss

```diff
@@ -46,12 +44,99 @@ input#url_original_url {
 
 .btn {
   border-radius: 25px;
+  background: rgba(209, 85, 233, 0.72);
+  border: #9516c7;
```

Comment:
Color literals like `#9516c7` should only be used in variable declarations; they should be referred to via variable everywhere else.